set.mm

```diff
@@ -670161,10 +670161,12 @@ is real and it is less than or equal to its inferior limit (in such a
      real numbers. $)
   clsxlim $a class ~~>* $.
 
-  $d x k $.
-  $( Define the convergence relation for extended real sequences.  (Contributed
-     by Glauco Siliprandi, 5-Feb-2022.) $)
-  df-xlim $a |- ~~>* = ( ~~>t ` ( ordTop ` <_ ) ) $.
+  ${
+    $d x k $.
+    $( Define the convergence relation for extended real sequences.
+       (Contributed by Glauco Siliprandi, 5-Feb-2022.) $)
+    df-xlim $a |- ~~>* = ( ~~>t ` ( ordTop ` <_ ) ) $.
+  $}
 
```

Comment:
There are only constants in this definition, @tirix what about simply removing

 $d x k $.

?  (and no extra scope needed)

Glauco

Reply:
You're totally right, thank you! I fixed that.